base/src/main/java/butter/droid/base/providers/media/VodoProvider.java

```diff
@@ -77,7 +79,22 @@ protected Call enqueue(Request request, com.squareup.okhttp.Callback requestCall
                         "Mozilla/5.0 (Linux; U; Android %s; %s; %s Build/%s) AppleWebkit/534.30 (KHTML, like Gecko) PT/%s",
                         Build.VERSION.RELEASE, LocaleUtils.getCurrentAsString(), Build.MODEL, Build.DISPLAY,
                         versionName)).build();
-        return super.enqueue(request, requestCallback);
+        Call call = super.enqueue(request, requestCallback);
+
+        synchronized (ongoingCalls) {
```

Comment:
Yes at the moment that are `OpenSubs` and also `TraktProvider` that has hardcoded data in it.